interface/themes/login_page.scss

```diff
@@ -33,7 +33,7 @@ body.login {
   margin-top: 75px;
   max-width: 785px;
   padding: 25px;
-  box-shadow: 0 30px 60px 0 rgba(0,0,0,0.3);
+  box-shadow: 0 30px 60px 0 rgba($black,0.3);
```

Comment:
removed the variable then put it back in. For my older code, we need to use the 0,0,0.

Reply:
I just learned something new in GitHub. How to make the above kind of comment.